lib/color_mask.py

```diff
@@ -12,20 +14,23 @@ def get_frame(cap, scaling_factor):
 
     return frame
 
-if __name__=='__main__':
+def run_mask():
     cap = cv2.VideoCapture(0)
     scaling_factor = 0.5
 
     # Iterate until the user presses ESC key
     while True:
         frame = get_frame(cap, scaling_factor)
-
+        hsv_max = colors.hsv_max(frame)
         # Convert the HSV colorspace
         hsv = cv2.cvtColor(frame, cv2.COLOR_BGR2HSV)
 
         # Define 'blue' range in HSV colorspace
-        lower = np.array([60,100,100])
-        upper = np.array([180,255,255])
+        #lower = np.array([60,100,100])
+        #upper = np.array([180,255,255])
```

Comment:
Instead of leaving commented code (which often can be more confusing than clarifying), what if we changed it to a comment about what sort of values go into the `lower` and `upper` values?